doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -335,6 +335,8 @@ nRF9160 samples
     * New command ``link search`` for setting periodic modem search parameters.
     * Added printing of modem domain events.
     * MQTT support for ``gnss`` command A-GPS and P-GPS.
+    * Application specific modem fault handler.
+      The modem fault handler halts application execution in case of a modem crash.
```

Comment:
```suggestion
    * An application-specific modem fault handler.
      The modem fault handler halts application execution in case of a modem crash.
```